packages/driver/src/cy/commands/agents.ts

```diff
@@ -147,15 +153,15 @@ export default function (Commands, Cypress, cy, state) {
   // to reset the counts + the sandbox
   Cypress.on('test:before:run', resetAndSetSandbox)
 
-  const wrap = function (ctx, type, agent, obj, method, count) {
+  const wrap = function (ctx, type, agent, obj, method, count?) {
     if (!count) {
-      count = (counts[type] += 1)
+      count = (counts![type] += 1)
```

Comment:
For other TS newbies curious what this `!` is doing: [Non-null assertion operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator)